libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.html

```diff
@@ -31,15 +31,32 @@
 <!-- Horizontal Filters -->
 <div *ngIf="horizontal" class="sds-filters--horizontal">
   <div class="horizontal-filters grid-row">
-    <formly-form
-      *ngIf="!dialogRef"
-      [form]="form"
-      [fields]="fields"
-      [options]="options"
-      [model]="model"
-      (modelChange)="onModelChange($event)"
-    >
-    </formly-form>
+    <div *ngFor="let field of fields">
+      <div
+        *ngIf="!field.hide"
```

Comment:
Why did we remove this **!dialogRef** condition, this is for the mobile behaviour, can you check the mobile view, it's not broken

Reply:
I just checked the horizontal filters, mobile view still works fine after this change.